src/components/ha-date-range-picker.ts

```diff
@@ -41,6 +41,10 @@ export class HaDateRangePicker extends LitElement {
 
   @property({ type: String }) private _rtlDirection = "ltr";
 
+  @property({ type: Boolean }) private hideTimePicker = false;
+
+  @property({ type: Boolean }) private minimal = false;
```

Comment:
Why having 2 properties instead of one ?

Reply:
This way you can create a normal sized date-range pickers wich do not allow time selection.
This may be useful for some other feature.